ui/src/app/features/collection-manager-subject-right/collection-manager-subject-right/collection-manager-subject-right.component.ts

```diff
@@ -52,6 +52,9 @@ export class CollectionManagerSubjectRightComponent implements OnChanges {
   @Output() deleteSelectedExamples = new EventEmitter<string[]>();
   @Output() selectExample = new EventEmitter<CollectionItem>();
   @Output() loadMore = new EventEmitter<CollectionItem>();
+  @Output() restartUploading = new EventEmitter<CollectionItem[]>();
+
+  constructor() {}
```

Comment:
this looks redundant

Reply:
uploading images freezes(stays on hold) if you delete an item during uploading, it was used to prevent it and restart uploading

Reply:
I meant empty constructor